.ci/jenkins/Jenkinsfile

```diff
@@ -59,8 +59,7 @@ pipeline {
             steps {
                 script {
                     getMavenCommand(kogitoRuntimesRepo)
-                        .skipTests(true)
-                        .withProperty('skipITs', true)
+                        .withProperty('quickly')
```

Comment:
If this `-Dquickly` is the new way to build quickly, standarized across all KIE projects folowing Quarkus example, that is great. Would you mind doing a quick why-to-use + how-to-use on the zulip optaplanner channel (so the optaplanner team knows) and adjust the README.adoc to mention it exists?

Reply:
This was taken from Quarkus and I think is only implemented on Kogito/Optaplanner.
Kogito doc was already updated: https://github.com/kiegroup/kogito-runtimes/blob/main/CONTRIBUTING.md#build
I can leave a quick message on Optaplanner Zulip channel about it.